tests/coreclr/p1/Makefile

```diff
@@ -0,0 +1,121 @@
+TOP=$(abspath ../../..)
+include $(TOP)/defs.mak
+
+APPBUILDER=$(TOP)/scripts/appbuilder
+CORECLR_COMMON=$(TOP)/tests/coreclr/
+TEST_RUNNER=$(CORECLR_COMMON)/test-runner.sh
+RUN_SINGLE_TEST=$(CORECLR_COMMON)/run-single-test.sh
+STAT=$(CORECLR_COMMON)/stat.sh
+
+APPNAME=corerun
+TEST_CASE= \
+baseservices/threading/paramthreadstart/ThreadStartUShort/ThreadStartUShort.dll max
+PACKAGE_PATH  = myst/bin/corerun
+ifdef STRACE
+OPTS += --strace
+endif
+OPTS += --app-config-path=config_256m.json
+
+all: ext2fs build-package
+
+appdir:
+	$(APPBUILDER) -i vtikoo/coreclr-tests:6.0-p1-release
+
+rootfs: appdir
+	$(MYST) mkcpio appdir rootfs
+
+ext2fs: appdir
+	$(MYST) mkext2 appdir ext2fs
+
+build-package: package.pem
+	$(MYST) fssig --roothash ext2fs > roothash
+	$(MYST) package-sgx --roothash=roothash package.pem config_256m.json
+
+custom-test-runs:
+	$(RUN_SINGLE_TEST) $(MYST_EXEC) config_256m.json 300 ext2 \
+	JIT/Regression/CLR-x86-JIT/V1-M09/b16102/b16102/b16102.dll 0
+	$(RUN_SINGLE_TEST) $(MYST_EXEC) config_256m.json 300 ext2 \
+	JIT/jit64/regress/vsw/102754/test1/test1.dll 0
+
+	# # This test returns error code for non-windows systems, by design.
+	$(RUN_SINGLE_TEST) $(MYST_EXEC) config_256m.json 300 ext2 \
+	Interop/ExecInDefAppDom/ExecInDefAppDom/ExecInDefAppDom.dll 111
+
+	# # Change CWD using mystikos config for the following tests to run successfully
+	sed -i '19 i \ \ \ \ "CurrentWorkingDirectory":"/coreclr-tests-all/Interop/COM/Reflection/Reflection",' config_1g.json
+	$(RUN_SINGLE_TEST) $(MYST_EXEC) config_1g.json 300 ext2 \
+	Interop/COM/Reflection/Reflection/Reflection.dll
+	sed -i '19d' config_1g.json
+	sed -i '19 i \ \ \ \ "CurrentWorkingDirectory":"/coreclr-tests-all/JIT/Regression/JitBlue/DevDiv_461649/DevDiv_461649",' config_1g.json
```

Comment:
What are the "sed" commands for? Why can't we put `CurrentWorkingDirectory` in the config file?

Reply:
This was added while enabling the p1 suite. cc: @RRathna @bodzhang